### PR TITLE
Added tensorflow plugin docs 

### PR DESCRIPTION
**Pull request by evalsocket**
- Added tensorflow plugin docs 

fix: https://github.com/flyteorg/flyte/issues/1785

### CODE REVIEW

File: rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      git clone https://github.com/flyteorg/flytesnacks.git
+
+2. Start the Flyte sandbox for testing.
+
+   .. code-block:: bash
+
+      flytectl sandbox start --source=./flytesnacks
+      flytectl config init
+
+3. Install the PyTorch Operator.
```
Comment (samhita-alla) [inline rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst:21]:
```suggestion
3. Install the TensorFlow Operator.
```

File: rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
+.. _deployment-plugin-setup-k8s-tensorflow-operator:
+
+Kubeflow Tensorflow Operator Plugin Setup
```
Comment (samhita-alla) [inline rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst:3]:
```suggestion
Kubeflow TensorFlow Operator Plugin Setup
```

File: rsts/deployment/plugin_setup/k8s/index.rst [hunk 3]
```diff
@@ -34,6 +34,15 @@ K8s Plugin Setup
...
     Guide to setting up the Spark Operator Plugin.
 
+    ---
+
+    .. link-button:: deployment-plugin-setup-k8s-tensorflow-operator
+       :type: ref
+       :text: Tensorflow Operator
+       :classes: btn-block stretched-link
+    ^^^^^^^^^^^^
+    Guide to setting up the Tensorflow Operator Plugin.
```
Comment (samhita-alla) [inline rsts/deployment/plugin_setup/k8s/index.rst:44]:
```suggestion
    Guide to setting up the TensorFlow Operator Plugin.
```

File: rsts/deployment/plugin_setup/k8s/index.rst [hunk 4]
```diff
@@ -34,6 +34,15 @@ K8s Plugin Setup
     ^^^^^^^^^^^^
     Guide to setting up the Spark Operator Plugin.
 
+    ---
+
+    .. link-button:: deployment-plugin-setup-k8s-tensorflow-operator
+       :type: ref
+       :text: Tensorflow Operator
```
Comment (samhita-alla) [inline rsts/deployment/plugin_setup/k8s/index.rst:41]:
```suggestion
       :text: TensorFlow Operator
```

File: rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+                 container_array: k8s-array
+                 tensorflow: tensorflow
+
+5. Upgrade the Flyte Helm release.
+
+   .. code-block:: bash
+
+      helm upgrade -n flyte -f values-tensorflow.yaml flyteorg/flyte --kubeconfig=~/.flyte/k3s/k3s.yaml
+
+6. (Optional) Build & Serialize the Tensorflow plugin example. (TODO: https://github.com/flyteorg/flyte/issues/1757)
```
Comment (samhita-alla) [inline rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst:58]:
```suggestion
6. (Optional) Build & Serialize the TensorFlow plugin example. (TODO: https://github.com/flyteorg/flyte/issues/1757)
```

File: rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+   .. code-block:: bash
+
+      cd flytesnacks
+      flytectl sandbox exec -- make -C cookbook/integrations/kubernetes/kftensorflow serialize
+
+7. Register the Tensorflow plugin example. (TODO: https://github.com/flyteorg/flyte/issues/1757)
```
Comment (samhita-alla) [inline rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst:65]:
```suggestion
7. Register the TensorFlow plugin example. (TODO: https://github.com/flyteorg/flyte/issues/1757)
```

File: rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+   .. code-block:: bash
+
+      flytectl register files https://github.com/flyteorg/flytesnacks/releases/download/v0.2.225/snacks-cookbook-integrations-kubernetes-kftensorflow.tar.gz --archive -p flytesnacks -d development
+
+8. Lastly, fetch the launch plan, create and monitor the execution.
```
Comment (samhita-alla) [inline rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst:71]: Before we merge this, I think we need to have a TensorFlow example. I can try working on it.

Comment (evalsocket) [reply rsts/deployment/plugin_setup/k8s/tensorflow_operator.rst:71]: Yes

**Comment by evalsocket** [comment]
@kumare3 is it ok to merge it without an example. We have 2 pr(Tensorflow & Bigquery) in a similar state

**Comment by kumare3** [comment]
I think this is fine, but we should have an issue to write the docs for this

**Review by samhita-alla — APPROVED** [review]

**Pull request merged**